Support for BTTV emotes

TheWaII opened the pull request:
Added support for BTTV emotes. Renamed file to maintain consistency. Changed message to "Load emotes (7TV, BTTV) so the user knows which emotes are being loaded.

Commit:
Added support for BTTV emotes. Renamed file to maintain consistency. Changed message to "Load emotes (7TV, BTTV) so the user knows which emotes are being loaded.

adamff-dev commented on src/scripts/emotes.js:
```diff
@@ -45,6 +46,37 @@ import { showNotification } from './ui';
+
+      if (!res.ok) {
+        showNotification('res not ok');
+        emoteMap = new Map();
```
Create two separate maps: one for 7TV emotes and another for BTTV emotes, so the script can handle emotes from both platforms if a streamer uses them

adamff-dev commented on src/scripts/emotes.js:
```diff
@@ -45,6 +46,37 @@ import { showNotification } from './ui';
+      const emotes = data.sharedEmotes || [];
+
+      if (emotes.length === 0) {
+        showNotification('no emotes found');
```
Remove this notification

adamff-dev commented on src/scripts/emotes.js:
```diff
@@ -45,6 +46,37 @@ import { showNotification } from './ui';
+      emotes.forEach((emote) => {
+        const emoteId = emote.id;
+        const imageType = emote.imageType;
+        const size = '1x';
```
Define the size outside the loop. Or as a constant in the script

adamff-dev commented on src/scripts/emotes.js:
```diff
@@ -45,6 +46,37 @@ import { showNotification } from './ui';
+      const res = await fetch(`${API_BTTV}/users/twitch/${userId}`);
+
+      if (!res.ok) {
+        showNotification('res not ok');
```
Remove this notification

adamff-dev commented:
Can you see them now?

Commit:
implemented the suggestions.

now fetches 'channelEmotes' aswell; this was missing in the first implementation and would result on only showing half of the BTTV emotes.

TheWaII commented:
I've implemented the changes! I left the comments unresolved in case you had anything else to add. :) @adamff-dev

TheWaII commented:
thanks and sorry. I saw the feedback too late!
